VERSION

```diff
@@ -1 +1 @@
-0.0.1
```

Comment:
Is the removed newline an accident?

Reply:
Apparently not. Is there any particular reason why you remove the newline in VERSION?

Reply:
It's because you used `version = builtins.readFile ./VERSION`, so the derivation name would be `${pname}-${version}`, which is why it should not have white space in it.

Reply:
Is there any particular reason for a newline?

Reply:
mostly that some editors automatically add newlines at the end, so I'd like every file to be able to deal with newlines.

Reply:
Okay, I will use `builtins.replaceStrings` for this.

Reply:
On second thought, maybe we should add `.editorconfig`?

Reply:
I moved your comment into a new issue.